app/controllers/privacy_policy_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PrivacyPolicyController < ApplicationController
+  def index
+  end
+end
```

Comment:
This is OK but perhaps we just rename to StaticPagesController and intend to use it for any static pages we present? 

`def privacy`, `def about`, etc..